Python/Product/PythonTools/PythonTools/Commands/SendToReplCommand.cs

```diff
@@ -81,15 +81,19 @@ class SendToReplCommand : Command {
                 ITextSnapshotLine targetLine = snapshot.GetLineFromPosition(selection.Start.Position);
                 var targetSpan = targetLine.Extent;
 
-                if (IsCellMarker(targetLine)) {
-                    while (targetLine.LineNumber < snapshot.LineCount - 1) {
-                        var nextLine = snapshot.GetLineFromLineNumber(targetLine.LineNumber + 1);
-                        if (IsCellMarker(nextLine)) {
-                            break;
+                for (int lineNo = targetLine.LineNumber; lineNo >= 0; --lineNo) {
```

Comment:
The comment 3 lines above is now slightly out of date, you should mention what this block is doing, and/or extract into a method that makes it clearer.